dist/immutable.js

```diff
@@ -3799,6 +3799,8 @@
     return EMPTY_ORDERED_SET || (EMPTY_ORDERED_SET = makeOrderedSet(emptyOrderedMap()));
   }
 
+  var UNSUPPORTED_OPS = ['map', 'reduce', 'filter', 'sort', 'groupBy', 'sortBy', 'reduceRight', 'every', 'some'];
```

Comment:
let me know if I've left anything out here or you think that some of these should be supported